lib/aurora_bootstrapper/table.rb

```diff
@@ -73,5 +98,21 @@ def export_statement( into_bucket: )
           OVERWRITE ON
       SQL
     end
+
+    def object_uploaded?(s3_client, bucket_name, object_key)
+      response = s3_client.put_object(
```

Comment:
it is surprising to me as a reader to see a side effect in a `*?` method. Can we move this into a class? like a 
```ruby 
class Notifier
  def initialize( client )
    @client = client
  end

  def push_state( table )
    @client.whagever #=> push update about table
  end
end
```